scripts/lib/qa_crowbarsetup-help.sh

```diff
@@ -12,6 +12,9 @@ function qacrowbarsetup_help
         Option to set variable MTU size or select Jumbo Frames for Admin and Storage nodes. 1500 is used if not set.
     want_raidtype (default='raid1')
         The type of RAID to create.
+    want_batch_dir (default='/root')
```

Comment:
`$scenario_dir` would be a better default here.  We don't want to encourage filling up `/root` with random crap.